### PR TITLE
Revscoring 2.3.0 updates

### PR DESCRIPTION
**Pull request by halfak**
Minor updates and models rebuilt for revscoring 2.3.0

### CODE REVIEW

**Comment by halfak** [comment]
I manually checked all of the fitness stats from each model to confirm that there were no surprising changes.

File: model_info/arwiki.damaging.md [hunk 1]
```diff
@@ -15,67 +15,67 @@ Model Information:
...
+	 - release: '4.9.0-8-amd64'
 	
 	Statistics:
-	counts (n=18637):
+	counts (n=18611):
 		label        n         ~True    ~False
 		-------  -----  ---  -------  --------
-		True       340  -->       14       326
-		False    18297  -->       18     18279
+		True       340  -->        1       339
```
Comment (adamwight) [inline model_info/arwiki.damaging.md:24]: This doesn't look good.  Aren't these the actual counts?  What happened to our damaging examples?

**Review by adamwight — CHANGES_REQUESTED** [review]

File: model_info/arwiki.goodfaith.md [hunk 2]
```diff
@@ -15,67 +15,67 @@ Model Information:
...
 	
 	Statistics:
-	counts (n=18637):
+	counts (n=18611):
 		label        n         ~True    ~False
 		-------  -----  ---  -------  --------
-		True     18548  -->    18547         1
-		False       89  -->       89         0
+		True     18522  -->    18500        22
+		False       89  -->       86         3
```
Comment (adamwight) [inline model_info/arwiki.goodfaith.md:25]: Still not many true negatives, but "better" than zero?

File: model_info/bnwiki.reverted.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 Model Information:
```
Comment (adamwight) [inline model_info/bnwiki.reverted.md:1]: Tangentially, in future work let's include filename and checksum of input data sets in model info.

File: model_info/cswiki.damaging.md [hunk 4]
```diff
@@ -15,67 +15,67 @@ Model Information:
...
-		  0.956   0.956
-	fpr (micro=0.492, macro=0.268):
+		  0.965   0.965
+	fpr (micro=0.227, macro=0.131):
 		  False    True
 		-------  ------
-		  0.514   0.023
-	roc_auc (micro=0.919, macro=0.919):
+		  0.236   0.026
+	roc_auc (micro=0.958, macro=0.957):
```
Comment (adamwight) [inline model_info/cswiki.damaging.md:71]: This model grew healthier 8D

File: model_info/enwiktionary.reverted.md [hunk 5]
```diff
@@ -15,14 +15,14 @@ Model Information:
...
+	 - release: '4.9.0-8-amd64'
 	
 	Statistics:
-	counts (n=91825):
+	counts (n=91796):
 		label        n         ~True    ~False
 		-------  -----  ---  -------  --------
-		True       385  -->       10       375
-		False    91440  -->        3     91437
+		True       384  -->       10       374
```
Comment (adamwight) [inline model_info/enwiktionary.reverted.md:24]: I wonder why this model is so hesitant to predict true?

File: model_info/arwiki.damaging.md (hunk 1 above)
Comment (halfak) [reply model_info/arwiki.damaging.md:24]: There is the same number of damaging examples.  The model just (randomly) decided to build more strictly for the "predicted" threshold.

File: model_info/hewiki.damaging.md [hunk 6]
```diff
@@ -15,67 +15,67 @@ Model Information:
...
+	 - release: '4.9.0-8-amd64'
 	
 	Statistics:
-	counts (n=17817):
+	counts (n=19852):
 		label        n         ~True    ~False
 		-------  -----  ---  -------  --------
-		True       581  -->       69       512
-		False    17236  -->       16     17220
+		True       987  -->      636       351
```
Comment (adamwight) [inline model_info/hewiki.damaging.md:24]: Great!  Looks like a new dataset—it's a keeper.

File: model_info/idwiki.reverted.md [hunk 7]
```diff
@@ -15,67 +15,67 @@ Model Information:
...
+	 - release: '4.9.0-8-amd64'
 	
 	Statistics:
-	counts (n=98898):
+	counts (n=98654):
 		label        n         ~True    ~False
 		-------  -----  ---  -------  --------
-		True       959  -->      405       554
-		False    97939  -->     1757     96182
+		True      2580  -->     1680       900
```
Comment (adamwight) [inline model_info/idwiki.reverted.md:24]: Do you know what happened here?  This is a big change in the trusted labels, but I don't think the data set has updated beyond normal wear and tear?  Is this a bug in the merge logic?

It damaged model health, so this is worth investigation.

File: model_info/iswiki.reverted.md [hunk 8]
```diff
@@ -15,67 +15,67 @@ Model Information:
...
-		  0.936   0.936
-	fpr (micro=0.413, macro=0.239):
+		  0.941   0.941
+	fpr (micro=0.267, macro=0.163):
 		  False    True
 		-------  ------
-		  0.447    0.03
-	roc_auc (micro=0.943, macro=0.942):
+		  0.287   0.039
+	roc_auc (micro=0.953, macro=0.952):
```
Comment (adamwight) [inline model_info/iswiki.reverted.md:71]: Looking good

File: model_info/kowiki.reverted.md [hunk 9]
```diff
@@ -15,67 +15,67 @@ Model Information:
...
-		  0.951   0.951
-	fpr (micro=0.692, macro=0.371):
+		  0.916   0.916
+	fpr (micro=0.541, macro=0.313):
 		  False    True
 		-------  ------
-		  0.726   0.016
-	roc_auc (micro=0.871, macro=0.87):
+		  0.565   0.061
+	roc_auc (micro=0.856, macro=0.856):
```
Comment (adamwight) [inline model_info/kowiki.reverted.md:71]: Seems unhealthier, why?

File: model_info/nowiki.reverted.md [hunk 10]
```diff
@@ -36,46 +36,46 @@ Model Information:
...
-		  0.978   0.978
-	fpr (micro=0.232, macro=0.127):
+		  0.977   0.977
+	fpr (micro=0.258, macro=0.141):
 		  False    True
 		-------  ------
-		  0.236   0.018
-	roc_auc (micro=0.966, macro=0.965):
+		  0.263   0.019
+	roc_auc (micro=0.96, macro=0.958):
```
Comment (adamwight) [inline model_info/nowiki.reverted.md:71]: Random, minor worseness?

File: model_info/arwiki.damaging.md (hunk 1 above)
Comment (adamwight) [reply model_info/arwiki.damaging.md:24]: Ah!  Thanks, I hadn't thought about this implication of the 50% threshold.  Maybe we should actually be more explicit about this sort of thing, and choose a default threshold e.g. 60% prediction for "true" which is designed to balance some operational outcome.  I'm not sure what target metric would make sense here.

It's crazy that there isn't a standard orientation for these tables.

File: model_info/arwiki.damaging.md [hunk 11]
```diff
@@ -15,67 +15,67 @@ Model Information:
...
-		  0.978   0.978
-	fpr (micro=0.938, macro=0.48):
+		  0.979   0.979
+	fpr (micro=0.976, macro=0.499):
 		  False    True
 		-------  ------
-		  0.959   0.001
-	roc_auc (micro=0.939, macro=0.939):
+		  0.997       0
+	roc_auc (micro=0.936, macro=0.936):
```
Comment (adamwight) [inline model_info/arwiki.damaging.md:71]: Gotcha, I see that the health is unchanged.

File: model_info/idwiki.reverted.md (hunk 7 above)
Comment (halfak) [reply model_info/idwiki.reverted.md:24]: It seems that our autolabeling script got better at catching potentially damaging edits.  This tends to have a minor effect on our performance statistics.  I wouldn't consider this a big change in model health as many of the statistics are sensitive to rates in the target class.

File: model_info/kowiki.reverted.md (hunk 9 above)
Comment (halfak) [reply model_info/kowiki.reverted.md:71]: It's a pretty minor change.  I bet it had something to do with changes in our autolabeler.

File: model_info/nowiki.reverted.md (hunk 10 above)
Comment (halfak) [reply model_info/nowiki.reverted.md:71]: Very minor.  Randomness probably.

File: model_info/enwiktionary.reverted.md (hunk 5 above)
Comment (halfak) [reply model_info/enwiktionary.reverted.md:24]: Very very low positive rate I think

**Pull request merged**